### PR TITLE
Update the ChangeLog.md

### PR DESCRIPTION
**Pull request by daxian-dbw**
@TravisEz13 need you to review the 'Compliance Work' section to see
1. shall we list the PR titles or just PR numbers
2. are they all compliance wo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,116 @@
 # Changelog
 
+## v6.0.0-rc - 2017-11-16
+
+### Breaking changes
+
+- Fix `-Verbose` to not override `$ErrorActionPreference`. (#5113)
+- Fix `Get-Item -LiteralPath a*b` to return error if `a*b` doesn't actually exist. (#5197)
+- Remove `AllScope` from most default aliases to reduce overhead on creating new scopes. (#5268)
+- Change `$OutputEncoding` to be `UTF8` without `BOM` rather than `ASCII`. (#5369)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: I think this should be: Change `$OutputEncoding` default to be ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,116 @@
...
+
+### Engine updates and fixes
+
+- Add char range overload to the `..` operator, so `'a'..'z'` returns characters from 'a' to 'z'. (#5026) (Thanks @IISResetMe!)
+- Remove `CommandFactory` because it serves no real purpose. (#5266)
+- Change to not insert line breaks to output (except for tables). (#5193)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Reword as: Change to not insert line breaks at console window width to output (except for tables).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,116 @@
...
+- Remove `DCOM` support from `*-Computer` cmdlets. (#5277)
+- Add multiple link header support to Web Cmdlets. (#5265) (Thanks @markekraus!)
+- Use wider columns for process id and user. (#5303)
+- Add `Remove-Alias` Command. (#5143) (Thanks @PowershellNinja!)
+- Update `installpsh-suse.sh` to work with the `tar.gz` package. (#5309)
+- Add `Jobject` serialization support to `ConvertTo-Json`. (#5141)
+- Display full help with 'help' function. (#5195) (Thanks @rkeithhill!)
+- Fix `help` function to not pipe to `more` if objects are returned instead of help text. (#5395)
+- Fix `Unblock-File` to not write an error if the file is already unblocked. (#5362) (Thanks @iSazonov!)
+- Add error on legacy credential over non-HTTPS for Web Cmdlets. (#5402) (Thanks @markekraus!)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: This should be breaking change

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,116 @@
...
+- Clean up FullCLR code from Web Cmdlets. (#5376) (Thanks @markekraus!)
+- Exclude cmdlets that are not supported on Unix platforms. (#5083)
+- Make `Import-Csv` support `CR`, `LF` and `CRLF` as line delimiters. (#5363) (Thanks @iSazonov!)
+- Fix spelling in Web Cmdlet errors. (#5427) (Thanks @markekraus!)
+- Add `SslProtocol` support to Web Cmdlets. (#5329) (Thanks @markekraus!)
+- Fix single value JSON `null` in `Invoke-RestMethod`. (#5338) (Thanks @markekraus!)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: This is breaking change

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,116 @@
...
+- Update `Find-Dotnet` to find dotnet with the compatible SDK. (#5341) (Thanks @rkeithhill!)
+- Add signing manifest and script to update it with production values. (#5397)
+- Add `install-powershell.ps1` to install PowerShell Core on windows. (#5383)
+- Make `-Name` a dynamic parameter in `Start-PSPackage`. (#5415)
+- Support `[package]` tag in PR CI and fix nightly build on macOS. (#5410)
+- Enhance `install-powershell.ps1` to work on Linux and macOS. (#5411)
+- Move the `RCEdit` step to the build phase rather than the packaging phase. (#5404)
+- Allow packaging from a zip package to allow for signing. (#5418)
+- Add automation to validate PowerShell Core packages using Docker containers. (#5401)
+- Try fixing the `brew update` issue in bootstrap script. (#5400)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Can we remove `Try` and just say `Fixed`?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,116 @@
...
+- Enable `install-powershell.ps1` to update the current running PowerShell Core. (#5429)
+- Add standard set of VSCode workspace setting files. (#5457) (Thanks @rkeithhill!)
+- Add support for installing PowerShell Core on Amazon Linux via `install-powershell.sh`. (#5461) (Thanks @DarwinJS!)
+- Get `PowerShellGet` and `PackageManagement` from the PowerShell Gallery. (#5452)
+- Fix `Start-PSBuild` on `WSL` if repository was already built on Windows. (#5346) (Thanks @bergmeister!)
+- Fix build in VSCode and use an improved version of `tasks.json`. (#5453)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: I think we should call out rkeithhill for this one since he provided tasks.json

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,116 @@
...
+- Change all links in `README.md` to absolute as they are being used in other places outside of GitHub. (#5354)
+- Update instructions to build on VSCode based on `pwsh`. (#5368)
+- Update `FAQ.md` about how to use PowerShell Core nuget packages. (#5366)
+- Correct the Fedora documentation (#5384) (Thanks @offthewoll!)
+- Add instructions about how to create the `PowerShell.Core.Instrumentation` nuget package. (#5396)
+- Updated PowerShell to use the latest help package. (#5454)
+
+### Compliance Work
+
+Some compliance works are done as required for Microsoft to continue to
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: I don't think we need to say `some` here, just: Compliance work is required for Microsoft to continue...

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,116 @@
...
+sign and release packages from the project as official Microsoft packages.
+
+- Add term rules. (#5213)
```
Comment (TravisEz13) [inline CHANGELOG.md]: combine with 107, just list two PRs

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,116 @@
...
+- Use context rules instead of file or file and context rules because path to file can very. (#5297)
+- Replace the word `hang` with something more appropriate (#5358)
+- Use simplified names for compliance folders (#5388)
```
Comment (TravisEz13) [inline CHANGELOG.md]: this is more just a build and packaging improvement.  line 110 is as well.

File: CHANGELOG.md (hunk 8 above)
Comment (TravisEz13) [reply CHANGELOG.md]: Also combine with 108, and say something like:
`Replace the word ``hang`` with something more appropriate and add rules about other terms`

**Comment by daxian-dbw** [comment]
@SteveL-MSFT @TravisEz13 Your comments are addressed. Please take another look. Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Will change.

File: CHANGELOG.md (hunk 2 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Will do.

File: CHANGELOG.md (hunk 3 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Moved to breaking changes.

File: CHANGELOG.md (hunk 4 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Moved to breaking changes.

File: CHANGELOG.md (hunk 5 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 6 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Changed to `... use an improved version of ``tasks.json`` from @ rkeithhill`

File: CHANGELOG.md (hunk 7 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Updated as suggested.

File: CHANGELOG.md (hunk 8 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Updated.

**Review by SteveL-MSFT — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,114 @@
...
+- Add `Jobject` serialization support to `ConvertTo-Json`. (#5141)
+- Display full help with 'help' function. (#5195) (Thanks @rkeithhill!)
+- Fix `help` function to not pipe to `more` if objects are returned instead of help text. (#5395)
+- Fix `Unblock-File` to not write an error if the file is already unblocked. (#5362) (Thanks @iSazonov!)
+- Clean up FullCLR code from Web Cmdlets. (#5376) (Thanks @markekraus!)
+- Exclude cmdlets that are not supported on Unix platforms. (#5083)
+- Make `Import-Csv` support `CR`, `LF` and `CRLF` as line delimiters. (#5363) (Thanks @iSazonov!)
+- Fix spelling in Web Cmdlet errors. (#5427) (Thanks @markekraus!)
+- Add `SslProtocol` support to Web Cmdlets. (#5329) (Thanks @markekraus!)
+- Add `PSTypeName` Support for `Import-Csv` and `ConvertFrom-Csv`. (#5389) (Thanks @markekraus!)
```
Comment (markekraus) [inline CHANGELOG.md]: Are we considering this one a breaking change? technically it is.. but... thew likelihood it actually affects anyone is pretty low.

**Review by joeyaiello — APPROVED** [review]
Every one of these that you guys do is better than the last. Ship it. :)

File: CHANGELOG.md (hunk 10 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: It turns out the issue #5134 is marked with 'Breaking-Change'. I will move this one to the breaking change section.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**